natapokie/ai-fashion-mirror
# adding prettier and eslint

## Description

**Fixes #2 **

Adding prettier and eslint for more consistent code formatting and better code practices with typesciript. For prettier, used .json file from orientation website repo (feel free to suggest other changes!)

See readme file in this pr for more info on how to configure your editor.

Make sure to also run `npm install` in the root directory to install dev dependencies needed for prettier and eslint.

## Documentation Used

- [installing eslint](https://www.npmjs.com/package/eslint)
- [installing prettier](https://prettier.io/docs/en/install#eslint-and-other-linters)
- [configure prettier with ide](https://prettier.io/docs/en/editors)
- [using eslint with prettier](https://github.com/prettier/eslint-config-prettier#installation)

## How To Test?

Once the editor is configured properly, you should see eslint warnings, e.g.,
![image](https://github.com/user-attachments/assets/1b4c8a84-f4d3-4779-99f6-b758d0098501)


On save, your code should also be autoformatted, e.g.,
`const myObj={field1:1, 'field2':2}` -> `const myObj = { field1: 1, field2: 2 };`
